geode-core/src/main/java/org/apache/geode/distributed/internal/InternalDistributedSystem.java

```diff
@@ -136,6 +136,12 @@
   public static final String DISABLE_MANAGEMENT_PROPERTY =
       DistributionConfig.GEMFIRE_PREFIX + "disableManagement";
 
+  /**
+   * Feature flag to enable multiple caches within a JVM.
+   */
+  public static boolean ALLOW_MULTIPLE_SYSTEMS =
```

Comment:
Any reason, this is not using SystemPropertyHelper?

Reply:
Hmm, I've never heard about that class before. I can change this PR to use that.